crates/xds/src/client.rs

```diff
@@ -338,9 +338,39 @@ pub(crate) struct DeltaClientStream {
 impl DeltaClientStream {
     #[inline]
     async fn connect(
-        mut client: AdsGrpcClient,
+        endpoints: &[Endpoint],
         identifier: String,
-    ) -> Result<(Self, tonic::Streaming<DeltaDiscoveryResponse>)> {
+    ) -> Result<(Self, tonic::Streaming<DeltaDiscoveryResponse>, Endpoint)> {
+        if let Ok((mut client, ep)) = MdsClient::connect_with_backoff(endpoints).await {
+            let (req_tx, requests_rx) =
+                tokio::sync::mpsc::channel(100 /*ResourceType::VARIANTS.len()*/);
+
+            // Since we are doing exploratory requests to see if the remote endpoint supports delta streams, we unfortunately
+            // need to actually send something before the full roundtrip occurs. This can be removed once delta discovery
+            // is fully rolled out
```

Comment:
I think we've rolled delta xDS long enough that is no longer needed. The client can assume delta.